src/gui/optionsdlg.cpp

```diff
@@ -1339,6 +1479,49 @@ void OptionsDialog::setLocale(const QString &localeStr)
     m_ui->comboI18n->setCurrentIndex(index);
 }
 
+void OptionsDialog::on_comboBoxColorTheme_activated(int index)
```

Comment:
Maybe somebody wants to re-apply a theme. Last time it was me tuning the Plasma colors.